frontend/src/lib/components/Frontpage.svelte

```diff
@@ -13,60 +13,10 @@ export let getStarted = "";
 
 export let items = [
 	{
-		header: "Was ist Mondey?",
-		summary:
-			"Mondey ist ein wissentschaftlich geprüftes Programm zure Dokumentation der Entwicklung von Kindern bis 6 Jahren.",
+		header: "Möchten Sie die Entwicklung von Kindern begleiten und fördern?",
```

Comment:
you can use the `svelte-i18n` package to make use of localization. There is currently one default localization file under `frontend/src/locales` which is called `de.json`. You can define a section "frontpage" there and add all the items you need: 

```json 
"frontpage": {
    "heading": "Möchten Sie die Entwicklung von Kindern begleiten und fördern?",
     "summary": "Hier sind Sie genau richtig!",
     "buttonLabel": "Anmeldung"
} 
```

which then can be used in the  `*.svelte` files like so: 

```javascript
import {_} from "svelte-i18n";

export let items = [
       {header: $_("frontpage.heading"), ... }
```
and so on. If you want to use these in the markup section of the component, you have to wrap it in `{}`.